src/views/GovernanceTaskList/index.test.tsx

```diff
@@ -55,25 +56,22 @@ describe('The Goveranance Task List', () => {
         </MemoryRouter>
       );
     });
-    setImmediate(() => {
```

Comment:
The use of `setImmediate` in tests was causing a lot of warnings to be printed about components updating outside `act` calls.

There is also a warning about tests being `async` and also taking a `done` callback- it's supposed to be one or the other now. Many tests were slightly restructured to remove the use of the `done` callback in order to address this.